test/createExamples.js

```diff
@@ -0,0 +1,76 @@
+        display: "standalone",                    // Preferred display mode: "fullscreen", "standalone", "minimal-ui" or "browser". `string`
+        orientation: "any",                       // Default orientation: "any", "natural", "portrait" or "landscape". `string`
+        scope: "/",                               // set of URLs that the browser considers within your app
+        start_url: "/?homescreen=1",              // Start URL when launching the application from a device. `string`
+        version: "1.0",                           // Your application's version string. `string`
+        logging: false,                           // Print logs to console? `boolean`
+        pixel_art: false,                         // Keeps pixels "sharp" when scaling up, for pixel art.  Only supported in offline mode.
+        loadManifestWithCredentials: false,       // Browsers don't send cookies when fetching a manifest, enable this to fix that. `boolean`
+        icons: {
+            android: true,              // Create Android homescreen icon. `boolean` or `{ offset, background, mask, overlayGlow, overlayShadow }`
+            appleIcon: true,            // Create Apple touch icons. `boolean` or `{ offset, background, mask, overlayGlow, overlayShadow }`
+            appleStartup: true,         // Create Apple startup images. `boolean` or `{ offset, background, mask, overlayGlow, overlayShadow }`
+            coast: true,                // Create Opera Coast icon. `boolean` or `{ offset, background, mask, overlayGlow, overlayShadow }`
+            favicons: true,             // Create regular favicons. `boolean` or `{ offset, background, mask, overlayGlow, overlayShadow }`
+            firefox: true,              // Create Firefox OS icons. `boolean` or `{ offset, background, mask, overlayGlow, overlayShadow }`
+            windows: true,              // Create Windows 8 tile icons. `boolean` or `{ offset, background, mask, overlayGlow, overlayShadow }`
+            yandex: true                // Create Yandex browser icon. `boolean` or `{ offset, background, mask, overlayGlow, overlayShadow }`
+        }
+    },
+    callback = function (error, response) {
+        if (error) {
+            console.log(error.message); // Error description e.g. "An unknown error has occurred"
+            return;
+        }
+
+        // Check if tmp dir exists, if not create
+        if (!fs.existsSync(__dirname + '/tmp/')){
+            fs.mkdirSync(__dirname + '/tmp/');
+        }
+
+        // Save images
+        response.images.forEach(item => {
+            fs.writeFile(__dirname + '/tmp/' + item.name, item.contents, 'binary', function(err){
+                if (err) throw err
+                console.log(item.name + ' saved.')
+            })
+        });
+
+        // Save files
+        response.files.forEach(item => {
+            fs.writeFile(__dirname + '/tmp/' + item.name, item.contents, 'binary', function(err){
```

Comment:
I'd also favor string interpolation here and pretty much everywhere else, where strings are explicitly concatenated by the `+` operator.